frontend/src/global_styles/content/work_packages/_table_content.sass

```diff
@@ -107,6 +107,9 @@
   z-index: 1
   padding: 0 !important
 
+  .generic-table--sort-header-outer
+    display: block
```

Comment:
It is somehow ugly that the configuration trigger needs a special treatment for display although there isn't anything special about it. That is an indicator that the solution is not optimal.